src/cmd/manage_nsfs.js

```diff
@@ -933,6 +927,10 @@ function validate_options_type_by_value(input_options_with_data) {
             if (['allow_bucket_creation', 'regenerate', 'wide', 'show_secrets'].includes(option) && validate_boolean_string_value(value)) {
                 continue;
             }
+            // special case for bucket_policy (from_file)
+            if (option === 'bucket_policy' && type_of_value === 'object') {
```

Comment:
so this adds a new type for bucket policy? now bucket policy is string | object?
If so, this should be added to consts & help file instead of as a special case here

Reply:
It is just for `from_file` option to accept another type, since we use a JSON file, we don't need it to be a string but can be an object.
In all cases, we set one type:
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/manage_nsfs/manage_nsfs_constants.js#L69-L90

But may accept more types in the function `validate_options_type_by_value` (`name`, boolean types that we accept `true` and `false`, etc.):
https://github.com/noobaa/noobaa-core/blob/d1bf8c465af745dedbff9d3803ec13da344884ba/src/cmd/manage_nsfs.js#L929-L935

Reply:
@shirady Yes, exactly the same as with the booleans/string booleans.
My point is that -
We have a list of constant types for the CLI arguments but it's not full, we add more special types on several cases on this function..
We need to consider refactoring this function since the missing types on the types list is confusing